scripts/deployment/globals_base_mainnet_meme_alpha.json

```diff
@@ -18,14 +18,14 @@
   "stakingTokenAddress":"0xEB5638eefE289691EcE01943f768EDBF96258a80",
   "stakingFactoryAddress":"0x1cEe30D08943EB58EFF84DD1AB44a6ee6FEff63a",
   "memeFactoryAddress":"0x82A9c823332518c32a0c0eDC050Ef00934Cf04D4",
-  "livenessRatio":"34722222222222",
-  "memeActivityCheckerAddress":"0x026AB1c5ea14E61f67d245685D9561c0c2Cb39Ba",
+  "livenessRatio":"347222222222222",
```

Comment:
should we update this with 1 heart per day?? If yew, we should redeploy activity checker